src/app/components/select/style/selectstyle.ts

```diff
@@ -197,8 +197,7 @@ input.p-select-label {
 
 .p-dropdown.ng-invalid.ng-dirty,
 .p-select.ng-invalid.ng-dirty {
-    outline: 1px solid ${dt('selectbutton.invalid.border.color')};
-    outline-offset: 0;
+    border-color: ${dt('selectbutton.invalid.border.color')};
```

Comment:
I believe the variable name is also wrong. Instead of "selectbutton" it should be "select".